orca-6/Dockerfile

```diff
@@ -117,5 +112,4 @@ wiggletools \
 wtdbg2 \
 xmatchview \
 xssp \
-yaha \
-vim
```

Comment:
Can you please add `vim` back to orca-0/Dockerfile?

Reply:
`vim` is there already :) This temporary addition was a hack because I had to uninstall ruby

Reply:
Never mind. `vim` is already in there.

Reply:
Jinx! Thanks, Lauren.